airflow-core/src/airflow/ui/src/layouts/Details/Gantt/Gantt.tsx

```diff
@@ -185,9 +187,9 @@ export const Gantt = ({ limit }: Props) => {
           minBarLength: MIN_BAR_WIDTH,
         },
       ],
-      labels: [],
+      labels: chartLabels,
```

Comment:
```suggestion
      labels: flatNodes.map((node) => node.id),
```

I think it would be better and more clear in this way.

Reply:
agree, I was thinking to get the labels specifically memoized would be performant but could be trivial, thanks and making the updates now.